.github/workflows/macOS_gcc10.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           dbus \
           discount \
           flex \
-          gnupg@2.2 \
+          gnupg \
```

Comment:
There is a PATH at line 61, please remove the whole line. (GitHub does not let me comment there 😕)

Also gnupg not needed explicitly here (same as above).